lib/highest_score.rb

```diff
@@ -13,9 +13,6 @@ def self.highest_score_from(array_of_words)
           wordscore_hash[w] = score
         end
 
-        #find max value of a value in a hash
-        # wordscore_hash.max[0]
-
         #returns all keys with max value
         top_scoring_words_hash = {}
         wordscore_hash.each do |k, v|
```

Comment:
Overall, I'd recommend using full `key` and `value` variable names, rather than the single-letter variables, `k` and `v`